cmd/riff/main.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 the original author or authors.
+ * Copyright 2018-2019 the original author or authors.
```

Comment:
this kind of change may look surprising (this specific file was created in 2019) but `headache` follows renames, hence the insertion of 2018.

Reply:
many of these date ranges are still wrong, listing the file as being first authored in 2018 when they were really created in 2019. Git tracks content, not files. Renames are only an estimation. During the rewrite, many of the files are listed as renames when they were really delete and add.

Reply:
Yep, I see your point. I don't think there is an easy fix for this problem but the least `headache` can do is warn whenever a rename is detected and let the user decide whether the date range should be kept or not. Closing the PR for now.